.buildkite/hooks/pre-command

```diff
@@ -3,7 +3,7 @@
 set -eu
 
 # Only execute in the verify pipeline
-[[ "$BUILDKITE_PIPELINE_NAME" =~ verify$ ]] || exit 0
+[[ "$BUILDKITE_PIPELINE_NAME" =~ verify$ ]] || [[ "$BUILDKITE_PIPELINE_NAME" =~ validate/release$ ]] || [[ "$BUILDKITE_PIPELINE_NAME" =~ validate/adhoc$ ]]  || [[ "$BUILDKITE_PIPELINE_NAME" =~ validate/adhoc-canary$ ]] || exit 0
```

Comment:
shouldn't you be able to just match on leading `'validate/'`

Reply:
yeah no problem, updated!